dashboard/app/models/course_version.rb

```diff
@@ -89,4 +95,9 @@ def destroy_and_destroy_parent_if_empty
   def contained_lessons
     units.map(&:lessons).flatten
   end
+
+  def all_standards_url
+    prefix = content_root_type === 'UnitGroup' ? 'courses' : 's'
+    "/#{prefix}/#{name}/standards"
+  end
```

Comment:
In the future, if we move standalone courses (like csf courses) to live on /courses/, then this logic could be simplified to always use the /courses/ prefix.